integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/NewCarbonDataLoadRDD.scala

```diff
@@ -425,9 +427,11 @@ class NewDataFrameLoaderRDD[K, V](
         TableProcessingOperations.deleteLocalDataLoadFolderLocation(model, false, false)
         // in case of failure the same operation will be re-tried several times.
         // So print the data load statistics only in case of non failure case
-        if (SegmentStatus.LOAD_FAILURE != loadMetadataDetails.getSegmentStatus) {
-          CarbonTimeStatisticsFactory.getLoadStatisticsInstance
-            .printStatisticsInfo(CarbonTablePath.DEPRECATED_PATITION_ID)
+        if (!carbonLoadModel.isCarbonNonTransactionalTable) {
```

Comment:
this check should not be present, logic should be same for transactional and non-transactional tables;

Reply:
Done